core/src/test/scala/org/apache/spark/shuffle/sort/SortShuffleManagerSuite.scala

```diff
@@ -130,5 +130,4 @@ class SortShuffleManagerSuite extends SparkFunSuite with Matchers {
       mapSideCombine = true
     )))
   }
-
```

Comment:
nit: revert the unrelated change.

Reply:
line-133 is empty line, we add test in  `SortShuffleManagerSuite ` before, after https://github.com/apache/spark/pull/31664#discussion_r585669990, move the test to `ShuffleSuite`, So i change the empty line together.

Reply:
I know the reason, but it's still not a necessary change, especially when there's no other changes in the file.

Reply:
ok，i will revert it.